python/python-psi-api/src/com/jetbrains/python/psi/FutureFeature.java

```diff
@@ -28,7 +28,7 @@ public enum FutureFeature {
   ABSOLUTE_IMPORT("absolute_import", LanguageLevel.PYTHON25, LanguageLevel.PYTHON30),
   PRINT_FUNCTION("print_function", LanguageLevel.PYTHON26, LanguageLevel.PYTHON30),
   UNICODE_LITERALS("unicode_literals", LanguageLevel.PYTHON26, LanguageLevel.PYTHON30),
-  ANNOTATIONS("annotations", LanguageLevel.PYTHON37, LanguageLevel.PYTHON311)
+  ANNOTATIONS("annotations", LanguageLevel.PYTHON37, LanguageLevel.PYTHON312)
```

Comment:
`__future__.annotations` is turned on in the 3.11 betas so far, but it's actually been delayed again, so shouldn't be in 3.11 release.

Reply:
- See this https://github.com/python/cpython/pull/93628